torch/csrc/jit/frontend/versioned_symbols.cpp

```diff
@@ -106,5 +108,42 @@ uint64_t get_min_version_for_kind(const NodeKind& kind) {
   return it->second;
 }
 
+uint64_t get_min_operator_version(const Module& module) {
+#if ENABLE_UPGRADERS
+  uint64_t min_version = caffe2::serialize::kMinSupportedFileFormatVersion;
+#else
```

Comment:
you can probably remove this macro now that its been a while we enabled this feature and got no issues so far.

Reply:
done.